install_config/install/advanced_install.adoc

```diff
@@ -516,14 +516,16 @@ xref:../../architecture/additional_concepts/networking.adoc#openshift-sdn[{produ
 SDN].
 
 However, in order to ensure that your masters are not burdened with running
-pods, you can make them
+pods, they are by default marked unschedulable.
 xref:../../admin_guide/manage_nodes.adoc#marking-nodes-as-unschedulable-or-schedulable[unschedulable]
-by adding the `*openshift_schedulable=false*` option any node that is also a
-master. For example:
+
+The examples in this document use masters as 'infra' nodes and have been marked
+schedulable ensuring the router and registry are deployed properly. In a production
+environment you should have dedicated infra nodes. For example:
```

Comment:
Eh, maybe I should move the 'For example:' bit around, this reads really poorly.